iib/web/models.py

```diff
@@ -1118,6 +1130,7 @@ def from_json(  # type: ignore[override] # noqa: F821
                 'deprecation_list',
                 'graph_update_mode',
                 'build_tags',
+                'check_related_images',
```

Comment:
Can I please ask why we are not adding `check_related_images` attribute to the database? Also, it's not present in `to_json` which means we won't be able to see it in the API response meaning we would probably have to check the logs to determine if the check was run.

Reply:
I did not add it to the database earlier as I think this is similar to a validation check, the output index image will not be impacted. 
I agree that the parameter should be seen in response as well, will update in next patch

Reply:
Updated. Please resolve the comment if the changes are satisfactory

Reply:
Since you're validating the attribute in the ReqestAdd `from_json` itself and this parameter is not going to be used by multiple request types, shouldn't we not pass `check_related_images ` while calling the `from_json` of the parent class?

Reply:
I tried not passing to from_json of the parent class, it errors with "The following parameters are invalid: check_related_images"

Reply:
Ah, I see. It's the generalized check for valid/invalid parameters. Thanks!